plugins/scaffolder/src/components/Router.tsx

```diff
@@ -34,15 +34,21 @@ type RouterProps = {
   TemplateCardComponent?:
     | ComponentType<{ template: TemplateEntityV1beta2 }>
     | undefined;
+  TaskPageComponent?: React.ReactElement | undefined;
```

Comment:
Think you can re-use the `ComponentType`, and just do `ComponentType<{}>` Rather than `React.ReactElement` just to keep the `api-report` a little cleaner.

Reply:
The Route's element param will not accept a ComponetType. I could wrap it in a ReactElement maybe before passing to the route in maybe?

Reply:
Ok, Ive done that. I think.